src/App.tsx

```diff
@@ -1,61 +1,113 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
```

Comment:
add your `types` folder to the commit 😉 
<img width="370" alt="image" src="https://user-images.githubusercontent.com/55802630/184602875-01c88f75-a4ce-4773-a441-4e5556e1ecef.png">